src/ert/plugins/hook_implementations/forward_model_steps.py

```diff
@@ -598,3 +620,57 @@ def documentation() -> Optional[ForwardModelStepDocumentation]:
 @plugin(name="ert")
 def installable_forward_model_steps() -> List[Type[ForwardModelStepPlugin]]:
     return [*_UpperCaseFMSteps, *_LowerCaseFMSteps]
+
+
+def _get_eclrun_env(config: Dict[str, Any]) -> Dict[str, str]:
+    env_path = os.getenv("PATH", "")
+    ecl_path = config.get("eclrun_env", {}).get("PATH")
```

Comment:
Why take in the entire config dict? Could take in only `ecl_path` as an arg

Reply:
Yes, we can take in only the `ecl_path` as an arg, that will simplify it a bit.